### PR TITLE
fix(note): remove default connotation based icon 

### PR DESCRIPTION
**Pull request by rachelbt**

### CODE REVIEW

File: libs/components/src/lib/note/README.md [hunk 1]
```diff
@@ -39,6 +39,21 @@ Note that if not set, icon will default to the connotation-associated icon type.
 <vwc-note icon="home"></vwc-note>
 ```
 
+
+### No-icon
```
Comment (yinonov) [inline libs/components/src/lib/note/README.md:43]: how about `icon="false | null | undefined"`?

**Review by yinonov — CHANGES_REQUESTED** [review]

File: libs/components/src/lib/note/README.md (hunk 1 above)
Comment (rachelbt) [reply libs/components/src/lib/note/README.md:43]: instead of a new member?

File: change/@vonage-vivid-50cc6185-d6dd-4292-a82a-5530d8954408.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "note no-icon option",
```
Comment (rachelbt) [inline change/@vonage-vivid-50cc6185-d6dd-4292-a82a-5530d8954408.json:3]:
```suggestion
  "comment": "remove default connotation based icon",
```

File: libs/components/src/lib/note/note.spec.ts [hunk 3]
```diff
@@ -32,13 +31,18 @@ describe('vwc-note', () => {
...
-		const iconElement = element.shadowRoot?.querySelector('.icon') as Icon;
-		const iconName = 'home';
-		element.icon = iconName;
+		const icon = 'user-line';
+		element.setAttribute('icon', icon);
 		await elementUpdated(element);
+		const iconElement = element.shadowRoot?.querySelector('vwc-icon');
+		expect(iconElement?.getAttribute('name')).toEqual(icon);
+		expect(element.icon).toEqual(icon);
+	});
```
Comment (YonatanKra) [inline libs/components/src/lib/note/note.spec.ts:40]: image

File: libs/components/src/lib/note/note.spec.ts [hunk 4]
```diff
@@ -32,13 +31,18 @@ describe('vwc-note', () => {
...
 
-		expect(iconElement instanceof Icon).toEqual(true);
-		expect(iconElement.name).toEqual(iconName);
+	it('should not render icon when no icon member is set', async function () {
```
Comment (YonatanKra) [inline libs/components/src/lib/note/note.spec.ts:42]:
```suggestion
	it(`Should remove the icon when "no-icon" is set`, async function () {
```

File: libs/components/src/lib/note/note.spec.ts (hunk 3 above)
Comment (rachelbt) [reply libs/components/src/lib/note/note.spec.ts:40]: mastering the  art of copy-paste from other components :)

File: libs/components/src/lib/note/note.spec.ts (hunk 4 above)
Comment (rachelbt) [reply libs/components/src/lib/note/note.spec.ts:42]: no - there's no "no-icon" attribute.
If consumer do not add icon then there's no icon

File: libs/components/src/lib/note/note.ts [hunk 5]
```diff
@@ -23,8 +23,12 @@ export class Note extends FoundationElement {
 	 * HTML Attribute: headline
 	 */
 	@attr headline?: string;
+	@attr({ mode: 'boolean', attribute: 'no-icon' }) noIcon = false;
+
+
```
Comment (yinonov) [inline libs/components/src/lib/note/note.ts:28]:
```suggestion
```

File: libs/components/src/lib/note/ui.test.ts [hunk 6]
```diff
@@ -18,7 +18,7 @@ test('should show the component', async ({ page }: { page: Page }) => {
 		''
 	);
 
-	page.setViewportSize({ width: 900, height: 720 });
+	page.setViewportSize({ width: 900, height: 840 });
```
Comment (rachelbt) [inline libs/components/src/lib/note/ui.test.ts:21]:
```suggestion
	page.setViewportSize({ width: 900, height: 720 });
```

**Review by yinonov — DISMISSED** [review]

**Review by yinonov — APPROVED** [review]

**Review by yinonov — APPROVED** [review]

**Pull request merged**